mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,318 @@
+The following is the structure used by watcher module :
+
+```go
+//Watch structure contains the watcher specific configurations
+type Watch struct {
+	Enable     bool        `yaml:"enable" json:"enable"`
+	Attributes []Attribute `yaml:"attributes" json:"attributes"`
+}
+
+//Attribute structure contains the name of the attribute along with the actions to be performed for this attribute
+type Attribute struct {
+	Name    string   `yaml:"name" json:"name"`
+	Actions []string `yaml:"actions" json:"actions"`
+}
+```
+
+
+ ### Data Converter
+ 
+  The responsibility of data-converter is the convert the readings from the bluetooth device into a format understood by KubeEdge and vice-versa.
+  The input for this module can be provided by the user using the configuration file. The user needs to provide different conversion methods for write operations
+  and read operations and as per the requirements of their device.
+ 
+ ### Controller
+ 
+ Controller exposes MQTT API's for CRUD operations for managing Actions, Schedules and  Watchers. The controller is also responsible for starting all the other 
+  modules initially.   
+ 
+ ## Usage
+ 
+ ### Configuration File
+ 
+ The user can give the configurations specific to the bluetooth device using configurations provided in the configuration  file.
+ The user can enable or disable modules through the configuration files,  
+ 
+ **Example:** Below is an example for the configuration file for the temperature sensor and I/O control of Texas Instruments CC2650 device,
+  Similar to this user can create their own configuration file.  
+ 
+         mqtt:
+           mode: 0       # 0 -internal mqtt broker  1 - external mqtt broker
+           server: tcp://127.0.0.1:1883 # external mqtt broker url.
+           internal-server: tcp://127.0.0.1:1884 # internal mqtt broker url.
+         device:
+           name: <device name>
+           id: <deviceID>         #deviceID received while registering device with the cloud
+         action-manager:
+           actions:          # Multiple actions can be added
+           - name: <name of the action>
+             enable: <true/false>
+             operation:
+               action: <Write/Read>
+               characteristic-uuid: <UUID of the characteristic example:- f000aa0304514000b000000000000000>
+               value: [1]              # value in bytes to be written by defaut if converter for this is not mentioned                  
+         scheduler :
+           schedules:
+           - enable: <true/false>
+             name: <name of schedule>
+             event-frequency: <time in milliseconds>
+             occurrence-limit: <number of times to be executed>            # if it is 0, then the event will execute infinitely
+             actions:                        
+             - <action name>
+             - <action name> 
+         watch:
```

Comment:
watch or watcher?

Reply:
changed the name to watcher in the code for consistency,